### DISCUSSION TITLE
workflow/computation/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/workflow#30] [General]
# workflow/computation/

https://shanenull.com/workflow/computation/

**Comment by shane0** [comment]
https://playgameoflife.com/

**Reply by shane0** [reply]
https://shanenull.com/stoicism/#computations

**Reply by shane0** [reply]
# Coriolis

The Coriolis force is a fictitious force that appears to act on a mass in motion within a rotating frame of reference, such as the Earth. It is not a true force in the sense of Newtonian mechanics but rather an apparent force that arises due to the rotation of the reference frame.

The magnitude of the Coriolis force (\(F_c\)) acting on an object can be calculated using the following formula:

\[
F_c = 2m v \Omega \sin(\phi)
\]

Where:
- \(m\) is the mass of the object,
- \(v\) is the velocity of the object relative to the rotating frame,
- \(\Omega\) is the angular velocity of the rotating frame (for Earth, \(\Omega \approx 7.2921 \times 10^{-5}\) radians per second),
- \(\phi\) is the latitude at which the object is located.

### Key Points:
1. **Direction**: The Coriolis force acts perpendicular to the direction of motion and to the axis of rotation. In the Northern Hemisphere, it deflects moving objects to the right, while in the Southern Hemisphere, it deflects them to the left.
2. **Dependence on Latitude**: The force is strongest at the poles (\(\phi = 90^\circ\)) and zero at the equator (\(\phi = 0^\circ\)), because \(\sin(\phi)\) varies from 0 to 1.
3. **Velocity Dependence**: The faster the object moves (\(v\)), the greater the Coriolis force.
4. **Mass Dependence**: The force is directly proportional to the mass of the object.

### Example:
If a 1 kg object is moving at 10 m/s at a latitude of 45 degrees on Earth, the Coriolis force would be:

\[
F_c = 2 \times 1 \, \text{kg} \times 10 \, \text{m/s} \times 7.2921 \times 10^{-5} \, \text{rad/s} \times \sin(45^\circ)
\]

\[
F_c \approx 2 \times 1 \times 10 \times 7.2921 \times 10^{-5} \times 0.7071
\]

\[
F_c \approx 1.03 \times 10^{-3} \, \text{N}
\]

This is a very small force, which is why the Coriolis effect is only noticeable over large distances and long timescales, such as in the movement of air masses in the atmosphere or ocean currents.

**Reply by shane0** [reply]
fun fact: there are no hurricanes along the equator because the coriolis effect there is nearly 0

https://smv.org/learn/blog/why-dont-hurricanes-happen-at-the-equator/
